repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -490,7 +490,7 @@ export const getSalesOfferPackageList = (
                 Name: {
                     $t: `${product.productName} - ${userPeriodTicket.passengerType} - ${salesOfferPackage.name}`,
                 },
-                Description: { $t: `${salesOfferPackage.description}` },
+                Description: { $t: `${salesOfferPackage.description ?? ''}` },
```

Comment:
Looking at how user's create SOP'S currently, there's no way of them adding a description. Is there any time when the description wont be undefined? And if so, can we just remove all mention of it?

Reply:
As discussed will make sure this gets raised separately